tests/test_board.py

```diff
@@ -0,0 +1,53 @@
+from context import src
+from src import board as Board
+from src import cell as Cell
+import unittest
+
+
+class TestBoardProperties(unittest.TestCase):
+
+    def test_create_empty_board(self):
+        empty_board = Board.Board(0, 0)
+        self.assertEqual(len(empty_board._board), 0)
+
+    def test_create_small_board(self):
+        small_board = Board.Board(2, 3)
+        self.assertEqual(len(small_board._board), 2)
+        self.assertEqual(len(small_board._board[0]), 3)
+        self.assertTrue(small_board.has_cell(Cell(0, 0)))
+        self.assertTrue(small_board.has_cell(Cell(1, 0)))
+        self.assertTrue(small_board.has_cell(Cell(0, 1)))
+        self.assertTrue(small_board.has_cell(Cell(1, 1)))
+        self.assertTrue(small_board.has_cell(Cell(0, 2)))
+        self.assertTrue(small_board.has_cell(Cell(1, 2)))
+
+    def test_neighbours_of_cell(self):
+        small_board = Board.Board(2, 3)
```

Comment:
I do not like the fact that this simpe test is already this long. 
One reason is that we cannot parametrise easily which makes this awkard, so seee pytest.
The other thing, i think it would be better to check directly that neighbd set (although you have list but see my other review for the find_neighbours method). i.e. checking that nbd(0,0) = {(0,0),(1,0),(1,1)}

Reply:
I reccomend looking at this link [here](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html) for how nice pytest can be, hopefully you get what I mean!

Reply:
Agreed, pytest adopted